opm/output/Cells.hpp

```diff
@@ -32,6 +32,10 @@ class SimulationDataContainer;
 namespace data {
 
     struct CellData {
+        CellData(std::string name, UnitSystem::measure dim, std::vector<double> data)
```

Comment:
Since it's being copied anyway passing the arugments by value is fine (the copy might be elided this way). Clearly communicates what we're doing too, which is a bonus in my opinion.